python/graphframes/graphframe.py

```diff
@@ -16,7 +16,7 @@
 #
 
 from pyspark import SparkContext
-from pyspark.sql import Column, DataFrame, functions as sqlfunctions, SQLContext
+from pyspark.sql import Column, DataFrame, functions as sqlFuncs, SQLContext
```

Comment:
No reason (maybe trying to make it shorter). I see it is not good. I'll change it back. Or would you prefer some of these?
- `from pyspark import sql` and just call e.g. `sql.functions.col` and  `sql.functions.sum`
- `from pyspark.sql.functions import col, sum as sqlSum` or `sql_sum`